ui/app/mainui/AppMain.qml

```diff
@@ -811,24 +811,20 @@ Item {
                 rptDraggedPreviews.y = drag.y
             }
             onExited: cleanup()
-            Rectangle {
-                id: dropRectangle
-
-                width: parent.width
-                height: parent.height
-                color: Style.current.transparent
-                opacity: 0.8
-
-                states: [
-                    State {
-                        when: dragTarget.enabled && dragTarget.containsDrag
-                        PropertyChanges {
-                            target: dropRectangle
-                            color: Style.current.background
-                        }
-                    }
-                ]
+            Loader {
+                active: dragTarget.enabled && dragTarget.containsDrag
+                width: active ? parent.width : 0
+                height: active ? parent.height : 0
+                sourceComponent: Rectangle {
+                    id: dropRectangle
+
+                    width: parent.width
+                    height: parent.height
```

Comment:
```suggestion
```
"If the size of the Loader is specified explicitly by setting the width, height or by anchoring, the loaded item will be resized to the size of the Loader."

Reply:
TIL